src/dbnode/topology/types.go

```diff
@@ -106,6 +106,9 @@ type Map interface {
 	// LookupHostShardSet returns a HostShardSet for a host in the map
 	LookupHostShardSet(hostID string) (HostShardSet, bool)
 
+	// LookupInitializingHostPair returns the initializing host bootstrapping from specific shard from leaving host.
+	LookupInitializingHostPair(hostID string, id uint32) (string, bool)
```

Comment:
is this function symmetric? ie can i provide either the returning or the leaving hostID as the args and it'll return the other? either way, can you update the arg names to indicate which one you're expecting. (your comment seems to indicate its only accepting the leavingHostID but i'm not sure how you intend it)